lib/phoenix/config.ex

```diff
@@ -70,9 +70,14 @@ defmodule Phoenix.Config do
   Useful to read a particular value at compilation time.
   """
   def from_env(otp_app, module, defaults) do
-    Keyword.merge(defaults, fetch_config(otp_app, module), &merger/3)
+    otp_app
+    |> fetch_config(module)
+    |> merge(defaults)
```

Comment:
Reversing the order of merge but calling it merge is potentially very confusing. Or we keep the order and swap arguments here (non-pipeable) or we call it something like `reverse_merge`. I prefer the former.

Reply:
Maybe name it `override` or `into_defaults`.